### PR TITLE
Implemented Tagging closes #116

### LINKED ISSUE #116: add tags to blogs
<a href="https://github.com/eunicekokor"></img></a> **Issue by [eunicekokor](https://github.com/eunicekokor)**
_Tuesday Nov 18, 2014 at 18:58 GMT_
_Originally opened as https://github.com/danrschlosser/eventum/issues/116_

----

& in the future, side panel w/ related posts if tags are the same.
also kind of dependent on if we have more content incoming!

### PR DESCRIPTION
**Pull request by EvanNGoldstein**

### CODE REVIEW

File: app/routes/blog.py [hunk 1]
```diff
@@ -27,6 +27,21 @@ def index():
                            previous_index=previous_index,
                            next_index=next_index)
 
+@blog.route('/blog/<tag>', methods=['GET'])
```
Comment (danrschlosser) [inline app/routes/blog.py]: can this be `/blog/tag/<tag>`

**Comment by danrschlosser** [comment]
Would you show a screenshot of how you're displaying tags on the website, and on Eventum?

**Comment by EvanNGoldstein** [comment]
screen shot 2015-02-21 at 5 28 56 pm

screen shot 2015-02-21 at 5 29 27 pm

**Comment by danrschlosser** [comment]
Will this break if the tags have spaces in them?

**Comment by EvanNGoldstein** [comment]
No, it works fine with spaces.

**Comment by EvanNGoldstein** [comment]
I just discovered you can add an empty tag. Fixing that now (should only take a minute or two)

**Comment by EvanNGoldstein** [comment]
Updated screenshot with comma separated list:
screen shot 2015-02-21 at 5 48 21 pm

**Pull request merged**